Lib/fontTools/ttLib/tables/D__e_b_g.py

```diff
@@ -0,0 +1,16 @@
+from . import DefaultTable
+import json
+
+class table_D__e_b_g(DefaultTable.DefaultTable):
+
+	def decompile(self, data, ttFont):
+		self.data = json.loads(data)
```

Comment:
can you please use four spaces? I know we aren't consistent elsewhere but for new modules I'd prefer to ditch the tabs